.github/workflows/trivy_scan.yml

```
@@ -5,6 +5,13 @@ on:
     # every day at 10am
     - cron: '0 10 * * *'
   workflow_dispatch:
+    inputs:
+      # this is useful for testing an arbitrary released version of Antrea
+      # by default, we will use the latest release (obtained using the Github API)
```

You mean the default is latest release, right?
```suggestion
      # This is useful for testing an arbitrary released version of Antrea.
      # By default, we will use the latest release (obtained using the Github API)
```